modules/test/playwright/tests/frontend-data-set-views-web/pages/ActionsPage.ts

```diff
@@ -14,12 +14,17 @@ export class ActionsPage {
 	readonly newActionButton: Locator;
 	readonly newActionForm: {
 		addIconButton: Locator;
+		confirmationMessageText: Locator;
+		headlessPermissionKeyText: Locator;
```

Comment:
Need to have a couple of `permissionKey` locators to deal with different actions. The one for headless actions is required.